tests/run.sh

```diff
@@ -1,11 +1,11 @@
-#!/bin/bash
+#!/usr/bin/env bash
 set -ue
 #set -x
 
 echo "## Testing empty stdin"
-diff -u <(./opam-ed "add bin [\"foo\"]" <&-) <(echo "bin: [\"foo\"]")
+diff -u <(opam-ed "add bin [\"foo\"]" <&-) <(echo "bin: [\"foo\"]")
```

Comment:
Hm couldn't it be confusing that the test script tests the installed version of the program and not the locally-built source you're working on ?
Maybe we could do something like `alias opam-ed="dune exec -- opam-ed` ?

Reply:
`dune runtest` includes `_build/install/bin` into the `PATH`, so it's only a problem if it's ran by hand, in which case running `dune exec -- ./run.sh` would do it too. I think it's better to give the options to test either the installed version or the local one.

Reply:
Hm alright this makes sense.